docs/blog/posts/2023/srl-elk.md

```diff
@@ -0,0 +1,986 @@
+---
+date: 2023-02-02
+tags:
+  - syslog
+  - sr linux
+  - elk
+  - k8s
+authors:
+  - azyablov
+---
+
+# Integrating SR Linux fabric with ELK stack
+
+In a not-so-distant past, manually extracting, parsing, and reading log files produced by network elements was standard practice for a sysadmin. With arcane piping of old-but-good `grep`, `awk` and `sed` tools one could swiftly identify a problem in a relatively large system. This was a viable approach for quite some time, but it became prey to a massive scale.
+
+Today's network infrastructures often count thousands of elements, each emitting its own log messages. Getting through a log collection of this size with CLI tools designed decades ago might not be the best tactic. Also, correlating logs from network elements with application logs might be impossible without software solutions built with such use cases in mind.
+
+In this post, we would like to cover how an SR Linux-powered DC fabric can be integrated with a modern logging infrastructure built with ElasticSearch/Logstash/Kibana stack to collect, handle and view logs from network elements.
+
+<!-- more -->
```

Comment:
@azyablov
check this intro that I refactored. I think that is what I meant by covering the high level goals of the post